test.js

```diff
@@ -17,6 +17,12 @@ function unref () {
   this.connector.stream.unref()
 }
 
+test.onFinish(function () {
+  db.quit(function () {
+    process.exit(0)
+  })
```

Comment:
This should not be needed. If it is, then you have a bad bug.

Reply:
Seems that some connections are not completely closed in test and without this tape test never exits. Suggestions?

Reply:
Find out which one is left open and close it.

Reply:
Fixed with latest merge on aedes-persistence, we need to create a new release there